rest/data/task.go

```diff
@@ -90,6 +90,15 @@ func (tc *DBTaskConnector) FindTasksByIds(ids []string) ([]task.Task, error) {
 	return ts, nil
 }
 
+func (tc *DBTaskConnector) FindOldTasksByID(id string) ([]task.Task, error) {
+	ts, err := task.FindOld(task.ByIDPrefix(id))
```

Comment:
Why does this just find old executions? Why not search both collections?

Reply:
I was under that impression that only the most recent execution is stored in tasks, and all the older ones are stored in old_tasks. I only want the old executions. Was this assumption wrong?

Reply:
That's correct, I didn't realize that the caller of this retrieved the new and old versions separately